dlp/dlp_snippets/main.go

```diff
@@ -57,6 +59,14 @@ func main() {
 	flag.Parse()
 
 	infoTypesList := strings.Split(*infoTypesString, ",")
+	var customDictionariesList []string
+	if *customDictionaryString != "" {
+		customDictionariesList = []string{*customDictionaryString}
+	}
+	var customRegexesList []string
+	if *customRegexesString != "" {
+		customRegexesList = strings.Split(*customRegexesString, ",")
```

Comment:
This might cause problems if someone has a comma in their regex. So, maybe only allow a single regex?

Reply:
Done.